CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * @iov/keycontrol: Add `Keyring.getAllIdentities` and `Keyring.getWalletByIdentity`
 * @iov/keycontrol: Add `UserProfile.getAllIdentities`
 * @iov/keycontrol: Add `.previewIdentity` to the `ReadonlyWallet`/`Wallet` interfaces
+* @iov/jsonrpc: Let `isJsonCompatibleDictionary` not accept non-simple objects
```

Comment:
Huh? Not really clear what this is until diving into the issues.

Reply:
Looking at tests, I assume you mean you accept objects and Uint8Array?

Reply:
Also name changed to `isJsRpcCompatibleDictionary`, right?

Reply:
Before `isJsonCompatibleDictionary` accepted Uint8Array, Date, RegEx, etc as value. Those objects are not JSON compatible.

Now we have two different things
- `isJsonCompatibleDictionary` (fixed to accept JSON only)
- `isJsRpcCompatibleDictionary` a new function that allows more data types and can consume a full transaction object.